test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smmodelparallel.py

```diff
@@ -29,13 +29,13 @@
 
 def validate_or_skip_smmodelparallel(ecr_image):
     if not can_run_smmodelparallel(ecr_image):
-        pytest.skip("Model Parallelism is supported on CUDA 11 on TensorFlow v2.3.1 or higher")
+        pytest.skip("Model Parallelism is supported on CUDA 11 on TensorFlow version no lower than v2.3.1 and lower than v2.7.0")
```

Comment:
Could you make this more concise, such as "Model Parallelism is supported on CUDA 11 or above on TensorFlow versions between 2.3.1 and 2.7.0"?